tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -154,3 +225,5 @@ def test_foo(cls):
 
 
 lazy_imports = LazyImporter  # pylint: disable=invalid-name
+sys.path_hooks.append(LazyImporter)
+sys.path.append("FAKE_PATH_TRIGGER")
```

Comment:
Why are you adding this ? Please add comments when you write something non trivial which might be confusing for readers.